sql/2020/19_Compression/distribution_of_text_compression_lighthouse.sql

```diff
@@ -0,0 +1,15 @@
+#standardSQL
+#Distribution of Text Based Compression Lighthouse Scores
+SELECT 
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.score") AS text_compression_score,
+  COUNT(0) AS num_pages,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages
+FROM 
+  `httparchive.lighthouse.2020_08_01_*` 
```

Comment:
Note that the Lighthouse dataset is mobile-only, but I think it's ok to be explicit in the results with the client=mobile field as a reminder. No changes needed, just FYI.

Reply:
got them. thanks.